src/components/new-request/search.vue

```diff
@@ -44,40 +43,49 @@
         </v-select>
       </v-col>
 
-      <!-- location (aka jurisdiction) -->
-      <v-col cols="12" md="4" lg="4" :class="{'px-3': !isMobile }">
-        <v-tooltip id="location-options-select"
-                   top
-                   content-class="top-tooltip"
-                   transition="fade-transition"
-                   :disabled="!location || location === 'BC' || isMobile">
+      <!-- display a dummy input box here when Jurisdiction and Entity Type are not shown -->
+      <v-col v-if="!showJurisdiction && !showEntityType" cols="12" md="6" class="py-0">
+        <v-text-field filled disabled />
```

Comment:
Sev, I wonder what the UX folks think about this. I was looking at the UXPin and I don't see the situation where we haven't selected anything yet. Do you think they are OK with this?

Reply:
Excellent, deep review question!

I looked at all the uxpins, and the "changes" that Janis proposed (empty input on right until you select something), and the code itself, and I thought this was a pretty clean way to do it. I also specifically asked Janis and Yui to look this over. For sure, this is an "interpretation" of the multiple requirements and I'll change it if they don't like it.

Reply:
Makes sense, thanks for clearing that up Sev! 

FWIW, I think this is pretty clean and nice.

Resolving.